### PR TITLE
Feature/IPEHR-202-stats-web-server

### PR DESCRIPTION
**Pull request by SealTV**

### CODE REVIEW

**Review by Setest — APPROVED** [review]

**Review by 1db1 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,12 @@ config.json
...
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+data/local.db
+data/storage/ac/5c/6f/ac5c6f71777989e5e33338eabc56fceafd944f08ecb0e985c66a721e8c64a8dc
+data/storage/test_1675247705186023000/1d/db/b1/1ddbb106191e6a79ee2cc69eba198721a4e1e3f2fd0c43a9eb15db0f3d95c9a0
+data/storage/test_1675247705186023000/68/b1/cd/68b1cdb2a109538c9ba1190aae1cde1a80e0f182ea0e79581f5644e9ac7a790b
+data/storage/test_1675247705186023000/8d/07/49/8d07499980627f827d537563e57dab9107e97bfecc4d56d2a0ae30039bb49f47
+data/storage/test_1675247705186023000/e2/b4/9f/e2b49fc80b2b902a1a2f2748e0cde35affaf7fba93258e2a05d6e222c0cfbeeb
+data/storage/test_1675247705186023000/ec/e9/1d/ece91da8d42092e09ecafcb1e988027ae371eaa258a386383a84b29c50930487
```
Comment (1db1) [inline .gitignore]: we can exclude the whole contents of data/storage directory

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -7,9 +7,33 @@ on:
...
+        cd ./src
+        go test -v ./pkg/...
+
+  integraion_tests:
+    name: Integration Tests
+    runs-on: ubuntu-latest
+    # Service containers to run with `runner-job`
+    services:
+      stat:
+        image: ghrs.io/bsn-si/ipehr-stat:develop
```
Comment (1db1) [inline .github/workflows/go.yml]: ghcr.io

**Review by 1db1 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (SealTV) [reply .gitignore]: fixed it

File: .github/workflows/go.yml (hunk 2 above)
Comment (SealTV) [reply .github/workflows/go.yml]: Righ. fixed it

**Review by 1db1 — APPROVED** [review]

**Pull request merged**